app/views/passwords/edit.html.haml

```diff
@@ -0,0 +1,13 @@
+%section.reset_password.container
+  .row
+    .col-sm-10.col-sm-offset-1
+      = form_tag "/passwords/#{@user.password_reset_token}", method: :patch do
```

Comment:
The URL is hardcoded I would try something like `password_path(@user.password_reset_token)`